experiments/AMIP/components/atmosphere/climaatmos.jl

```diff
@@ -298,64 +359,6 @@ function calculate_surface_air_density(atmos_sim::ClimaAtmosSimulation, T_S::Cli
     extrapolate_ρ_to_sfc.(Ref(thermo_params), ts_int, swap_space!(ones(axes(ts_int.ρ)), T_S))
```

Comment:
`calculate_surface_air_density` one is required

Reply:
Ah we missed this in the docs - I'll update them

Do we need to require both `calculate_surface_air_density` and `get_field(:air_density)`?

Reply:
update: as discussed offline, these two functions return different values, which the docs have been updated to reflect